package/kernel/i2c-aw9523-expander/src/i2c-leds-aw9523.c

```diff
@@ -0,0 +1,269 @@
+// SPDX-License-Identifier: GPL-2.0+
```

Comment:
Is this driver upstream to the linux kernel? If not, please do so and backport the driver. We are doing out best to reduce the amount of custom driver/patches.

Reply:
I don's see any reason to upstream this stub driver for unknown Taiwan IC found in cheapest Chinese router specially make for Russian 4G LTE operator.

Reply:
@mkresin @LynxChaus Looks like aw9523.c driver has been merged into a Linux kernel :smiley: Please go ahead